developer-docs-site/docs/concepts/basics-accounts.md

```diff
@@ -2,42 +2,107 @@
 title: "Accounts"
 id: "basics-accounts"
 ---
-An account represents a resource on the Aptos Blockchain that can send transactions. Each account is identified by a particular 32-byte account address and is a container for Move modules and Move resources.
 
-# Introduction
+# Accounts
 
-The state of each account comprises both code and data:
+An account on the Aptos Blockchain contains blockchain assets. These assets, for example, coins and NFTs, are by nature scarce and must be access-controlled. Any such asset is represented in the blockchain account as a **resource**. A resource is a Move language primitive that emphasizes on access control and scarcity in its representation of an asset.  
```

Comment:
This is true but also not the whole truth, resources aren't necessarily assets, e.g. would you consider a vote to be an asset. Still for an introduction this is still good, but perhaps we need an acknowledgment that resources aren't just assets.

emphasizes on -> emphasizes